cypress/e2e/po/pages/agent-configuration-rke2.po.ts

```diff
@@ -0,0 +1,232 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import UnitInputPo from '~/cypress/e2e/po/components/unit-input.po';
+import SelectPo from '~/cypress/e2e/po/components/select.po';
+import RadioGroupInputPo from '~/cypress/e2e/po/components/radio-group-input.po';
+
+function _parseArea(area:string) {
+  let areaId: string;
+
+  if (area === 'cluster') {
+    areaId = 'clusteragentconfig';
+  } else if (area === 'fleet') {
+    areaId = 'fleetagentconfig';
+  } else {
+    throw new Error(`Area identifier not recognized ::: ${ area }`);
+  }
+
+  return areaId;
+}
+
+export default class AgentConfigurationRke2 extends PagePo {
```

Comment:
I've moved this to be a `componentPo` since it's a component.  I've added the `nameNsDescription` which was the only bit that was missing

Reply:
AgentConfiguration is a component in the page for creating a cluster. This new component PO has functions to return the page title and the page component NameNsDescroption, both outside the scope of this component.

We already have a PO for the create rke2 cluster page (`cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke2-custom.po.ts`) which inherits functions to return page title and NameNsDescription. This new AgentConfiguration PO should be added tjere , and anything that needs something page like 


cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke2-custom.po.ts
```
  agentConfiguration(): AgentConfigurationRke2 {
    return new AgentConfigurationRke2(this.self())
  }
```

whereever this will be used...
```
const page = new ClusterManagerCreateRke2CustomPagePo()

page.title();

page.nameNsDescription().name().set(....;

page.agentConfiguration().clickTab(....
```

Reply:
updated